### DISCUSSION TITLE
Change category url from `/categories/category/` to `/category/`

### DISCUSSION THREAD

**Discussion by arunism** [ppoffice/hexo-theme-icarus#1159] [General]
Suppose I have to categories `Blog` and `Lifestyle`. By default if I navigate to categories and then to each one of those I get `/categories/blog/` and `categories/lifestyle` respectively. But I wanted something like `/blog/` and `/lifestyle/`.

If anyone has done this already, please do educate me with the way.

**Comment by ppoffice** [comment]
@arunism Set `category_dir` to `''` in your blog's `_config.yml`.

https://hexo.io/docs/configuration#Directory

**Reply by arunism** [reply]
already solved it but thanks
